assets/styles/_base.scss

```diff
@@ -107,12 +107,17 @@ body {
   background-color: $color-background-hero;
 
   &,
-  #{$heading-selectors} {
+  #{$heading-selectors},
+  a,
+  .feather {
     color: $color-background;
   }
 
-  .feather {
-    color: $color-background;
+  @media (min-width: $breakpoint-tablet) {
+    a:hover,
+    a:focus {
```

Comment:
Perhaps nesting the states inside a single `<a>` tag selector would be better. 
Like so:
```
a {
    &:hover,
    &:focus {
      ...
    }
}
```

Reply:
I didn't like the nesting and decided against it because I don't expect needing any styles on the a tag and thus I feel it wouldn't add value.